api/proto/config/devconfig.proto

```diff
@@ -121,6 +121,11 @@ message EdgeDevConfig {
 
   // A list of bond interfaces (LAGs) aggregating physical network adapters.
   repeated BondAdapter bonds = 31;
+
+  // Dormant time configured from controller. This decides the duration to
+  // wait between metrics messages before marking an edge-node as inactive
+  // in the controller.
+  string config_dormant_time = 32;
```

Comment:
Why do we need this in config?
We already have the configItem called timer.metric.interval and today the dormant time is 3x that timer by default.

What is missing is the dormant time in the info message.

Reply:
Removed from Config.